packages/gasket-utils/lib/index.d.ts

```diff
@@ -49,15 +49,11 @@ export function tryRequire(path: string): object|null;
  * @param context - Context for applying overrides
  * @param context.env - Name of environment
  * @param [context.commandId] - Name of command
- * @param [context.root] - Project root; required if using localeFile
- * @param [context.localFile] - Optional file to load relative to gasket root
  * @returns config
  */
-export function applyConfigOverrides(config: GasketConfig, { env, commandId, root, localFile }: {
+export function applyConfigOverrides(config: GasketConfigDefinition, { env, commandId }: {
```

Comment:
`commandId` still relevant?

Reply:
Dunno yet - it might still be so leaving until we make a final decision